src/components/uploadPicture/uploadImage.tsx

```diff
@@ -113,34 +115,23 @@ const UploadImageComponent: React.FC<UploadImageComponentProps> = ({
   return (
     <div className={styles.uploadContainer}>
       {currentStep === UploadStep.Start && (
-        <button
-          onClick={onClickStartButton}
-          disabled={isUploading}
-          className={styles.actionButton}
-        >
-          <CameraIcon />
-          {title}
-        </button>
-      )}
-      {currentStep === UploadStep.TakePicture && (
-        <>
+        <button className={styles.actionButton} onClick={onClickStartButton}>
           <input
             ref={fileInputRef}
             type="file"
             accept="image/*;capture=camera"
             capture="environment"
             onChange={handleFile}
-            className={styles.actionButton}
+            style={{ display: "none" }}
```

Comment:
a little late now but wondering if it'd be possible to style the input directly instead of wrapping with a button?

Reply:
Input type files tend to not like styling. It would be slightly better to use a label instead, making the onClick function unecessary, but the principle remains the same. https://stackoverflow.com/a/25825731